web/template/layout/page.gotmpl

```diff
@@ -65,6 +65,7 @@
                     sirius-pahub
                 {{ end }}
             {{ end }}
+            govuk-frontend-supported
```

Comment:
Is this meant to be here?

Reply:
This was the tag that we need to make sure the JS worked